cli/cmd/generate_aws.go

```diff
@@ -67,8 +67,9 @@ var (
 	// AwsArnRegex original source: https://regex101.com/r/pOfxYN/1
 	AwsArnRegex = `^arn:(?P<Partition>[^:\n]*):(?P<Service>[^:\n]*):(?P<Region>[^:\n]*):(?P<AccountID>[^:\n]*):(?P<Ignore>(?P<ResourceType>[^:\/\n]*)[:\/])?(?P<Resource>.*)$` //nolint
 	// AwsRegionRegex regex used for validating region input; note intentionally does not match gov cloud
-	AwsRegionRegex  = `(af|ap|ca|eu|me|sa|us)-(central|(north|south)?(east|west)?)-\d`
-	AwsProfileRegex = `([A-Za-z_0-9-]+)`
+	AwsRegionRegex     = `(af|ap|ca|eu|me|sa|us)-(central|(north|south)?(east|west)?)-\d`
+	AwsProfileRegex    = `([A-Za-z_0-9-]+)`
+	AwsAssumeRoleRegex = `^arn:(aws|aws-us-gov|aws-cn):iam::\d{12}:role\/.*$`
```

Comment:
Why include the `aws-us-gov` and `aws-cn` partitions?

Are you sure that `aws-us-gov` works with the Terraform?  Last time I looked, all our Terraform is hardcoded to the `aws` partition.

Some time back I was told by product that Lacework isn't supporting `aws-cn` partition.

Reply:
Good call. Removed `aws-us-gov` and `aws-cn`.